builtin.c

```diff
@@ -807,6 +810,13 @@ Perl_boot_core_builtin(pTHX)
     }
 
     newXS_flags("builtin::import", &XS_builtin_import, __FILE__, NULL, 0);
+
+    inc_hv = GvHVn(PL_incgv);
+    hv_store(inc_hv, "builtin.pm", STRLENs("builtin.pm"), newSVpvs(__FILE__), 0);
+    ver_gv = gv_fetchpvs("builtin::VERSION", GV_ADDMULTI, SVt_PV);
+    ver_sv = GvSV(ver_gv);
+    /* Remember to keep $VERSION in this file and $VERSION in builtin.pm synced. */
```

Comment:
Is it possible to have a test for that?

Reply:
the test is in this patch already. Change either side to 0.017 and test fails.

Reply:
Apologies, I missed that.